util/crc32c_arm64.cc

```diff
@@ -46,33 +52,59 @@
 extern bool pmull_runtime_flag;
 
 uint32_t crc32c_runtime_check(void) {
-#if !defined(__APPLE__)
+#if defined(__linux__) || defined(__FreeBSD__)
   uint64_t auxv = 0;
 #if defined(ROCKSDB_AUXV_GETAUXVAL_PRESENT)
   auxv = getauxval(AT_HWCAP);
 #elif defined(__FreeBSD__)
   elf_aux_info(AT_HWCAP, &auxv, sizeof(auxv));
 #endif
   return (auxv & HWCAP_CRC32) != 0;
```

Comment:
Can we move this code to the `#else` branch so it executes in case of non-Apple, non-OpenBSD?